samples/estimation/estimation-openqasm.ipynb

```diff
@@ -0,0 +1,609 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Getting Started with QDK Resource Estimation using OpenQASM\n",
+        "\n",
+        "👋 Welcome to the QDK Resource Estimator using OpenQASM. In this notebook we will\n",
+        "guide you how to estimate and analyze the physical resource estimates of a\n",
+        "quantum program targeted for execution based on the architecture design of a\n",
+        "fault-tolerant quantum computer. As a running example we are using a multiplier."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Implementing the algorithm"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "As a first step, we will create a sample application which will be used throughout this Resource Estimation notebook. To start, we'll import some utilities from the `qiskit` Python package eventually generating the OpenQASM program."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "tags": [
+          "parameters"
```

Comment:
What is this metadata/tags/parameters for? I think that's why I'm seeing the below when I open the notebook

<img width="583" alt="image" src="https://github.com/user-attachments/assets/448ae29f-54cc-409b-a6d7-701f01d2b7aa" />